### PR TITLE
Migrate README from Markdown to reStructuredText

### LINKED ISSUE #33: Pypi can't render README.md properly
No logo, no proper links, stars in headers.
https://pypi.org/project/rohmu/
image

According to [this](https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/#creating-a-readme-file) doc we should use [GitHub Flavored Markdown](https://github.github.com/gfm/) or [CommonMark](https://commonmark.org/)

### PR DESCRIPTION
**Pull request by axeoman**
Pypi having hard time rendering unflavoured Markdown, so an easiest
way to fix this would be to migrate to reStructuredText

Hope it will fix #33

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+PostgreSQL, MySQL, M3 and ClickHouse are trademarks and property of
+their respective owners. All product and service names used in this
+website are for identification purposes only and do not imply
+endorsement.
+
+Credits
+=======
+
+Rohmu was created by, and is maintained by,
```
Comment (alexole) [inline README.rst:62]: Could you please remove redundant commas here?

Comment (alexole) [reply README.rst:62]: I did not check line by line, but those commas where in the previous version of the test I guess. Hopefully nothing changes expect for this?

**Review by alexole — DISMISSED** [review]

File: README.rst (hunk 1 above)
Comment (axeoman) [reply README.rst:62]: Oh no, this is done by a crappy converter. Going to check for more of those.

Comment (axeoman) [reply README.rst]: Now checked line by line. Can't see any differences between the old and the new one.

**Review by alexole — APPROVED** [review]

**Pull request merged**